### ISSUE TITLE
Squiz/EmbeddedPhp doesn't examine code using short open echo tag

### ISSUE THREAD

**Issue by jrfnl** [squizlabs/PHP_CodeSniffer#3705]
Given this code:
```php
<?php   echo $var   ?>
```

Running the `Squiz.PHP.EmbeddedPhp` sniff over it results in:
```
-------------------------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------
 1 | ERROR | [x] Expected 1 space after opening PHP tag; 3 found
   |       |     (Squiz.PHP.EmbeddedPhp.SpacingAfterOpen)
 1 | ERROR | [x] Inline PHP statement must end with a semicolon (Squiz.PHP.EmbeddedPhp.NoSemicolon)
 1 | ERROR | [x] Expected 1 space before closing PHP tag; 3 found
   |       |     (Squiz.PHP.EmbeddedPhp.SpacingBeforeClose)
-------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------------
```

... while running the same sniff over the below code doesn't yield any errors:
```php
<?=   $var   ?>
```

### Questions
1. Should the `T_OPEN_TAG_WITH_ECHO` token be added to the sniff ?
2. If so and as the sniff would now start acting on more files, should the error codes for issues found in embedded PHP statements using short open echo tags be different ?

**Comment by jrfnl** [comment]
Closing as replaced by PHPCSStandards/PHP_CodeSniffer#27

**Comment by jrfnl** [comment]
FYI: a fix for this issue is included in today's [PHP_CodeSniffer 3.9.0 release](https://github.com/PHPCSStandards/PHP_CodeSniffer/releases/tag/3.9.0).
